firmware/application/apps/ui_recon.cpp

```diff
@@ -963,10 +959,9 @@ void ReconView::on_statistics_update(const ChannelStatistics& statistics) {
         last_timer = timer;
         text_timer.set("TIMER: " + to_string_dec_int(timer));
     }
-    if (timer) {
-        if (!continuous_lock || recon_match_mode == RECON_MATCH_SPARSE) {
-            timer -= time_interval;
-        }
+
+    if (timer != 0) {
```

Comment:
Are you ok with this going negative?

Reply:
I clip it at the only place it could ;-)
Could make it a uint one day, maybe not.